### ISSUE TITLE
Monetdb JDBC Driver Query Returned Null values

### ISSUE THREAD

**Issue by oscarspaz** [serge-rider/dbeaver#2037]
Dbeaver: 4.1.2
MonetDB JDBC Driver: monetdb-jdbc-2.6.jar

MonetDB Query Example

> SELECT sys.timestamp_to_str(TIMESTAMP '2016-02-04 15:30:29', '%Y%m-%d') AS date;

Result:

>     date |
>     -----|
>          | 

I was expecting the result to be '2016-02-04' instead of Null value. I tried using the same JDBC driver with Squirrel SQL Client and it returns a date value.

Also there is a Dbeaver warning:
> org.jkiss.dbeaver.model.exec.DBCException: SQL Error [0A000]: Operation getCharacterStream() currently not supported.

**Comment by serge-rider** [comment]
Well, MonetDB returns result of sys.timestamp_to_str as CLOB.
MonetDB JDBC driver doesn't support standard way to read CLOBs (getCharacterStream) - that's why you see empty value.
Although their JDBC driver supports another way (getSubString).

I'll add support of this function in DBeaver.

**Comment by palmaresk8** [comment]
I realized that this issue has the same cause as issue https://github.com/serge-rider/dbeaver/issues/2113 posted by me today. I also posted an issue in the MonetDB JDBC dev forum claiming for the implementation of getCharacterStream() method, and they answered that it will be implemented in the next release (https://www.monetdb.org/bugzilla/show_bug.cgi?id=6400). Besides that, it's curious how DBeaver and others GUI tools work differently in the same context. This could be an issue for other drivers too, not just MonetDB.
